research/picai/README.md

```diff
@@ -19,6 +21,30 @@ poetry install --with "picai"
 This will setup an environment with the proper dependencies to run the provided scripts out of the box. For more information about environment configuration, please refer to the [documentation](/CONTRIBUTING.md).
 
 ## Data
+### Important Definitions
+Below are the terms we will use to define model outputs at various stages of
+post-processing. They are ordered sequentially from least to most processed
+
+- **Logits:** The outputs of the model <ins>prior</ins> to the activation function. Values are unconstrained (-inf, inf)
+- **Probabilities:** The outputs of the model <ins>after</ins> a normalizing activation function such as softmax or sigmoid. Values are constrained to the range (0, 1). An example for a 2d image is shown below.
+
+  ![alt text](images/probs.png)
+- **Detection Maps:** Model predictions that contain an arbitrary number of distinct detected segmentation volumes derived from the output probabilities. Values are constrained to range [0, 1]. Example for a 2d image is shown below Detected segmentation volumes are defined as:
+  - Each detected volume is a connected component that must be non-connected and non-overlapping (mutually exclusive) with other volumes of the same class. (Therefore detection maps for multiclass segementation must be one hot encoded)
+  - Each pixel/voxel within a volume must have the same predicted probability. Therefore there is a single confidence/likelihood score for each volume.
+  - Detected segmentation volumes typically have a minimum size.
+  - The [report guided annotation](https://github.com/DIAGNijmegen/Report-Guided-Annotation) is a common api used for deriving detection maps from model output probabilities
+
+  ![alt text](images/detmap.png)
+- **Predicted Annotations/Segmentations**: Model predictions that have been thresholded to contain only class labels. If one hot encoded they must be binary {0, 1} or boolean {False, True} tensors. If not one hot encoded they must be be tensors containing only integers that represent the class labels (eg. constrained to {0, 1, 2, ...}). An example of a binary or one-hot-encoded predicted annotation is shown below
+
+  ![alt text](images/annotation.png)
+
+Below are some terms we will use to differentiate how models are combined
+
+- **Merging:** Merging models refers to combining the weights of several different models in some way to obtain a single model that can make predictions on the same data as the original models. This is distinct from federated learning in the sense that no training is traditionally done in this regime.
```

Comment:
nit: "can make predictions on the same data as the original models" maybe we say "format of data" just to highlight we aren't implying it has to be the exact same data